moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -315,6 +315,7 @@ export const useColumns = ({ hiddenColumns }) => {
       {
         headerName: "Substantial completion date",
         field: "substantial_completion_date",
+        description: "The earliest confirmed start date of a project phase that has a simple phase name of Complete (Complete or Post-construction)",
```

Comment:
🙌 @chiaberry @tillyw Could y'all coordinate for someone to make this description into a variable to hold this string so we can reference it in this PR and in https://github.com/cityofaustin/atd-moped/pull/1456? We may end up adding this in the Data Dictionary too so it could help us update the definition if we ever need to in the future.

I looked in the codebase, and maybe we could add a new file in `src/constants` to hold tooltip text. What do y'all think?

Reply:
I'll add this to my branch, trying to decide what to name the file, `projects.js` sound okay?

Reply:
I did it here: https://github.com/cityofaustin/atd-moped/pull/1456/commits/a60747a304869b88970d5fea2db0458c69820cba

Reply:
thanks, @chiaberry!

Reply:
@chiaberry sorry i merged your entire branch into this one when i should have just merged this one commit! i'll just make sure to merge this one last out of the substantial completion date PRs

Reply:
@tillyw sounds good! i'll hold off on approving this one since there are pending changes in Chia's PR. Thanks for planning ahead on the merge order since I brought Chia's issue into the mix late in the sprint. 🙏

Reply:
@mddilley all is in order! this one is ready to merge once i get your approval :)

Reply:
@tillyw awesome - thanks for getting this one caught up to main branch! 🙏

Could you update this line to use the variable that Chia created in https://github.com/cityofaustin/atd-moped/pull/1456/files though? I'll approve asap after that. Thank you!!

Reply:
i'm sorry i did not think before i rerequested review on this >.< will fix

Reply:
@mddilley imported!